client/my-sites/domains/domain-management/email/index.jsx

```diff
@@ -66,13 +66,7 @@ class Email extends React.Component {
 				</Header>
 			);
 		}
-		return (
-			<UpgradesNavigation
-				path={ this.props.context.path }
-				cart={ this.props.cart }
-				selectedSite={ this.props.selectedSite }
-			/>
-		);
+		return <PlansNavigation cart={ this.props.cart } path={ this.props.context.path } />;
```

Comment:
Would be nice to update the method name too, in order for it to reflect the new component name (`headerOrUpgradesNavigation` to something like `headerOrPlansNavigation`).

Reply:
👍 028fd669a1fcf9fc92476538b67dbdeebce7ef19